docs/concepts/send.md

```diff
@@ -0,0 +1,20 @@
+# SEND script
+
+A bash script that transfers the generated data (as a .tar.gz archive) to the target machine.
```

Comment:
bash/perl/python, ... whichever

it needn't pass .tar.gz at all - it can call external API for example